ui-tests/src/test/resources/features/integrations/integration-import-export.feature

```diff
@@ -95,13 +85,15 @@ Feature: Integration - Import Export
     And sleep for jenkins delay or "3" seconds
     # start integration and wait for published state
     And start integration "Integration_import_export_test"
+    Then check visibility of "Integration_import_export_test" integration details
+
     And navigate to the "Integrations" page
+    And Integration "Integration_import_export_test" is present in integrations list
+
+    And sleep for "300000" ms
```

Comment:
Please try manually importing this integration to both minishift and openshift and tell us how long it takes to:
1. build the integration
2. deploying the integration
3. time from starting the integration till UI is informed that the integration is in running state

Do the same but create the integration from scratch instead of importing it and compare the results.

Create a notes document and keep it up to date with your findings - we may find some performance issues thanks to your work.

Reply:
Here are tables with times:
[https://docs.google.com/spreadsheets/d/1I5FkFjHASaWzfeiI1tAqcyk33qqd2lzmHUALwew5yCc/edit?usp=sharing](url)